src/hypergraph/viz/instructions.py

```
@@ -0,0 +1,690 @@
+        if actual_target:
+            instructions.add_edge(input_node_id, actual_target)
+
+    # 2. Add edges between function nodes (from graph structure)
+    for source, target, edge_data in flat_graph.edges(data=True):
+        # Skip if either node is not visible
+        if not _is_node_visible(source, flat_graph, expansion_state):
+            continue
+        if not _is_node_visible(target, flat_graph, expansion_state):
+            continue
+
+        edge_type = edge_data.get("edge_type", "data")
```

Ruff (0.14.14)

[error] Local variable `edge_type` is assigned to but never used

Remove assignment to unused variable `edge_type`

(F841)